Makefile

```diff
@@ -347,9 +347,10 @@ androidupdate: all
 	# in runtime luajit-launcher's libluajit.so will be loaded
 	-rm $(INSTALL_DIR)/koreader/libs/libluajit.so
 
-        # assets are compressed manually and stored inside the APK.
+  # assets are compressed manually and stored inside the APK.
```

Comment:
Oh ffs GH online editor. :man_facepalming:  Anyway, still to be tested; I don't have Android stuff on this laptop.